tests/unit/backend/corpora/api_server/test_v1_collection.py

```diff
@@ -863,3 +863,54 @@ def test__update_collection_links__OK(self):
 
         self.assertEqual(200, response.status_code)
         self.assertEqual(links, json.loads(response.data)["links"])
+
+
+class TestCollectionsCurators(BasicAuthAPITestCurator):
+    def test_view_non_owned_private_collection__ok(self):
+        # Generate test collection
+        collection = self.generate_collection(
+            self.session, visibility=CollectionVisibility.PRIVATE.name, owner="another_test_user_id"
+        )
+
+        headers = {"host": "localhost", "Content-Type": "application/json", "Cookie": get_auth_token(self.app)}
+        test_url = furl(path=f"/dp/v1/collections/{collection.id}", query_params=dict(visibility="PRIVATE"))
+        response = self.app.get(test_url.url, headers=headers)
+
+        # This will pass even for non curators.
+        # Why are users allowed to view private collections that they don't own?
```

Comment:
Is this a todo question?

Reply:
Yes, if you look at the code [here](https://github.com/chanzuckerberg/single-cell-data-portal/blob/main/backend/corpora/lambdas/api/v1/collection.py#L39), there is no explicit filter for this endpoint that requires the user to be an owner. Only tombstone datasets and `access_type` are gated. @MDunitz do you know if this is as expected?

Reply:
At the time of implementation we decided that private datasets shouldnt be displayed on the portal page but they should be available to anyone with the link to handle cases where the owner of the dataset wanted to share it with other collaborators